src/core/model/Setsquare.cpp

```diff
@@ -27,7 +30,7 @@ auto Setsquare::getToolRange(bool transformed) const -> Range {
         rg.addPoint(-h, 0);
         rg.addPoint(0, h);
     }
-    rg.addPadding(.5 * xoj::view::SetsquareView::LINE_WIDTH);  // account for line width
+    rg.addPadding(.5 * xoj::view::SetsquareView::LINE_WIDTH * CM);  // account for line width
```

Comment:
Could you make the unit of SetsquareView::LINE_WIDTH explicit? e.g. add a comment at the declaration point, and maybe rename it to LINE_WIDTH_IN_CM

Reply:
I renamed it. A comment doesn't look necessary with the renamed constant.